region/maridia/outer.json

```diff
@@ -1095,6 +1128,141 @@
                   ],
                   "note": "This can be done with only a door-frame runway in the adjacent room.",
                   "devNote": "It can technically be done without a stutter, but it's not really easier."
+                },
+                {
+                  "name": "G-Mode Morph IBJ Overload Speed Blocks Frozen Crab (From the Middle)",
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "Gravity",
+                    "canTrickyUseFrozenEnemies",
+                    "Wave",
+                    {"or":[
+                      "h_canArtificialMorphJumpIntoIBJ",
+                      "canDoubleBombJump",
+                      "canStaggeredIBJ",
+                      "canBePatient",
+                      {"enemyKill": {
+                        "enemies": [ [ "Skultera" ] ],
+                        "explicitWeapons": [ "PowerBomb" ]
+                      }}
+                    ]},
+                    {"or":[
+                      "SpringBall",
+                      "h_canArtificialMorphCeilingBombJump",
+                      "canBePatient"
```

Comment:
The first `or` is for climbing the room, so double bomb jump helps there. The second `or` is for the tediousness of placing ~40 bombs against the speed blocks. `double bomb jump` would help here, but I still think it would be over 3 minutes without a ceiling IBJ or spring ball.

That is true. I'm hoping the strats here that dont require patience can be done before the crab gets there, but that is a good backup.

Reply:
I'm actually going to remove the spring ball option. I think youll miss the crab and need to wait if you cant ceiling bomb jump

Reply:
grav- springball 34, jump morph 42, dbj 90, springball and bouncing on your bombs 25, cibj 20
hijump - springball 30, jump morph 30

It helps a lot to stand directly below the speed blocks and jump straight up and down.
Its easier to get double bombs per cycle suitless.
double bomb jumps are tedious when going for this many and I don't like it.